src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Domain Models/ApprenticeshipCourse/FrameworksView.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.ProviderCommitments.Domain_Models.ApprenticeshipCourse
+{
+    public class FrameworksView
+    {
+        public DateTime CreatedDate { get; set; }
+        public List<Framework> Frameworks { get; set; }
```

Comment:
I think this would be better as Framework[] rather than List<Framework>.
If a class exposes a collection as a List then the class is explicitly saying that it is fine with consumers of that class adding and removing items from the list. An array doesn't allow that (with re-assigning the whole array). I don't think the view is expecting the collection to be amended by it's consumers.

Reply:
Addressed by https://github.com/SkillsFundingAgency/das-providercommitments/pull/8/commits/2823d9902e3267c6f292f0df08068083d8862add